CHANGELOG.md

```diff
@@ -15,7 +15,9 @@ the release.
   ([#2162](https://github.com/open-telemetry/opentelemetry-demo/pull/2162))
 * [cartservice] Refactor OpenFeature integration and add Dependency Injection support
   ([#2160](https://github.com/open-telemetry/opentelemetry-demo/pull/2160))
-
+* [chore] bump dependent images
+  ([#2179](https://github.com/open-telemetry/opentelemetry-demo/pull/2179))
+  
```

Comment:
Please remove trailing space.
Keep the line break, but remove the space 😅

Reply:
@julianocosta89 updated and resolved conflicts 😄